frontend/app/pods/admin/teams/template.hbs

```diff
@@ -7,6 +7,7 @@
 <div class="row">
   <div class="small-5 columns">
     {{#each model as |team|}}
+      {{#if team.name}}
```

Comment:
I've actually added this here because sometimes it opens up a nameless team temporary, which lives only in the ember.store without it going to the db or something. I do this so there won't be any empty entries. if you take it off, like in the users section, each time you press "add" for a new user or team, and then go back - you see an empty entry. only after a refresh it goes away. 
but no, you can't have a nameless team. this was made only for these kind of situations.